src/stream/src/task/stream_manager.rs

```diff
@@ -667,6 +668,7 @@ impl LocalStreamManagerCore {
                 .map(|b| b.try_into())
                 .transpose()
                 .context("failed to decode vnode bitmap")?;
+            let captured_context = actor.captured_context.clone().unwrap();
```

Comment:
There's `env` available here. Is it necessary to pass it from meta?

Reply:
Although also names as `StreamEnvironment`, it's actually [another structure](https://github.com/risingwavelabs/risingwave/blob/1fb029df7e517bd4611f2b0bc62f73641a20137a/src/stream/src/task/env.rs#L34) that does not contains `timezone`. [The one](https://github.com/risingwavelabs/risingwave/blob/1fb029df7e517bd4611f2b0bc62f73641a20137a/src/meta/src/model/stream.rs#L65C25-L65C25) contains timezone is never passed to stream manager.

Reply:
Good catch. 🤣 What about passing the real env here?

Reply:
🧐I prefer to use `CaptureExecutionContext` for several reasons:
1. We can unified the`Context` in Local, Batch and Streaming mode. If we need to add more besides `time_zone`, we just need to modify `CaptureExecutionContext`. 
2. If we pass `StreamEnvironment ` to streaming side, there will be two `env: StreamEnvironment` in one library but they are unrelated. 
3.  With `CaptureExecutionContext`, we can reuse some codes for local and distributed mode.

Reply:
Shall we rename `StreamEnvironment` to avoid the ambiguity?  

By the way, propose a new `ExprContext` as a substitute for `CapturedExecutionContext` because the `CapturedContext` is designed for expression (it's in the `expr` crate `src/expr/core/src/captured_execution_context.rs`)

Reply:
Not sure whether renaming the `StreamEnvironment` will cause a compatibility problem. Will take a try with name `StreamingContext`.

Do you mean renaming `CapturedExecutionContext ` to `ExprContext `? It soulds better than the current long name.

Reply:
> Do you mean renaming `CapturedExecutionContext` to `ExprContext`? 

Yes, 'cause it's long and everywhere

Reply:
The compatibility of renaming of `StreamEnvironment ` seems OK. Will open another PR to carry it out.

Renaming `CapturedExecutionContext` to `ExprContext` done.